examples/visualization/brain.py

```diff
@@ -0,0 +1,65 @@
+"""
+.. _ex-brain:
+
+===============================
+Plotting with ``mne.viz.Brain``
+===============================
+
+In this example, we'll show how to use :class:`mne.viz.Brain`.
+"""
+# Author: Alex Rockhill <aprockhill@mailbox.org>
+#
+# License: BSD-3-Clause
+
+# %%
+
+import os.path as op
+import matplotlib.pyplot as plt
+
+import mne
+from mne.datasets import sample
+
+print(__doc__)
+
+data_path = sample.data_path()
+sample_dir = op.join(data_path, 'MEG', 'sample')
+subjects_dir = op.join(data_path, 'subjects')
+evoked = mne.read_evokeds(op.join(sample_dir, 'sample_audvis-ave.fif'))[0]
+trans = mne.read_trans(op.join(sample_dir, 'sample_audvis_raw-trans.fif'))
+stc = mne.read_source_estimate(op.join(sample_dir, 'sample_audvis-meg'))
+
+brain = mne.viz.Brain('sample', subjects_dir=subjects_dir,
+                      alpha=0.5, cortex='low_contrast')
+
+# add sensors
+# brain.add_sensors(
+#     epochs.info, trans, colors=dict(eeg='green', grad='blue', mag='purple'),
+#     ch_types=dict(meg=True, eeg=True))
+
+# mark a region
+brain.add_label('BA3a', hemi='lh')
+
+# rotate brain by setting the camera position
+brain.show_view(azimuth=50, elevation=80, distance=250)
+
+# add a head
+# brain.add_head(color='red', alpha=0.1)
+
+# take a screenshot
+img = brain.screenshot()
+fig, ax = plt.subplots()
+ax.imshow(img)
+ax.axis('off')
+fig.suptitle('Brain')
+
+# %%
+# add an stc
+brain = mne.viz.Brain('sample', subjects_dir=subjects_dir)
```

Comment:
```suggestion
# %%
# Add source information
# ----------------------
#
# Plot source information.

sample_dir = op.join(data_path, 'MEG', 'sample')
evoked = mne.read_evokeds(op.join(sample_dir, 'sample_audvis-ave.fif'))[0]
trans = mne.read_trans(op.join(sample_dir, 'sample_audvis_raw-trans.fif'))
stc = mne.read_source_estimate(op.join(sample_dir, 'sample_audvis-meg'))
stc.crop(0.09, 0.1)

brain = mne.viz.Brain('sample', subjects_dir=subjects_dir, background='w')
```

Reply:
Why cant we build on the brain from above? It would be nice to show an image at the end that has the brain, sensors, region of interest, and source analysis all in one figure. Thoughts?

Reply:
I don't think you can show the an ROI colored in with an stc, they are both colorings of parts of the brain...

Reply:
Oh nevermind, I forgot about borders!

Reply:
yep.  Also I think the color used in `brain.add_label` can have an alpha, so depending on your STC's colormap it might sometimes even make sense to add a filled label on top of an STC.